src/modules/privacy-idea-administration/privacy-idea-administration.controller.ts

```diff
@@ -149,4 +151,36 @@ export class PrivacyIdeaAdministrationController {
             );
         }
     }
+
+    @Get('required')
+    @HttpCode(HttpStatus.OK)
+    @ApiOkResponse({
+        description: 'The requirement was successfully returned.',
+        type: TokenRequiredResponse,
+    })
+    @ApiBadRequestResponse({ description: 'A username was not given or not found.' })
+    @ApiUnauthorizedResponse({ description: 'Not authorized to get requirement information.' })
+    @ApiForbiddenResponse({ description: 'Insufficient permissions to get requirement information.' })
+    @ApiNotFoundResponse({ description: 'Insufficient permissions to get requirement information.' })
+    @ApiInternalServerErrorResponse({ description: 'Internal server error while getting requirement information.' })
+    @Public()
```

Comment:
Should not be public, since it uses PersonPermissions (Would also be the case for `getTwoAuthState`, although you didn't touch that one in this PR)